### PR TITLE
Fixes a regex error (item/rat`var`/something)

### LINKED ISSUE #8: var/ highlight is wrongly detected in the middle of typepath
image
----
image

Actual codeblock
------
```DM
/obj/item/weapon/ratvar/proc/do_something()
 return
```

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
* Closes #8

image

The image is the error that it catches a wrong detection about var

I added a regex condition for var `\b`

### CODE REVIEW

**Comment by PJB3005** [comment]
"People still use Atom?" oh wait this is still used by GitHub isn't it.

File: grammars/dream maker.cson [hunk 1]
```diff
@@ -25,7 +25,7 @@ patterns: [
 	}
 	{
 		match: '''(?x)
-		(var)[\\/ ]
+		(?:^|[\\s\\t\\(])(var)[\\/ ]
```
Comment (PJB3005) [inline grammars/dream maker.cson]: Wouldn't it make more sense/be sufficient to use `\w`?

Comment (EvilDragonfiend) [reply grammars/dream maker.cson]: some schizos can possibly use non-English as typepath name(although it's not a thing). `\s` `\t` `\(` `being first` would be all cases where `var/` is used.

Comment (PJB3005) [reply grammars/dream maker.cson]: I meant `\b`, sorry.

Comment (EvilDragonfiend) [reply grammars/dream maker.cson]: you're right. \b is cleaner.

**Comment by EvilDragonfiend** [comment]
* Changed the regex cleaner.
* Removed the incorrect increment for capture groups.

**Pull request merged**